betka/core.py

```diff
@@ -311,11 +311,15 @@ def sync_to_downstream_branches(self, branch) -> bool:
             return False
 
         # git {add,commit,push} all files in local dist-git repo
-        Git.git_add_all(
+        git_status = Git.git_add_all(
             upstream_msg=self.upstream_message,
             related_msg=Git.get_msg_from_jira_ticket(self.config),
         )
-
+        if not git_status:
+            self.info(
+                "There were no changes in repository. Do not file a pull request."
+            )
+            return False
```

Comment:
You are returning specific values True/False from this method but you do not use those values anywhere. It might make sense to do just `return` which returns None.

Reply:
Thanks for the comment. Let me check it.